test/e2e/common_test.go

```diff
@@ -292,15 +292,17 @@ func (p *PodmanTestIntegration) CreateArtifact(image string) error {
 	if os.Getenv("NO_TEST_CACHE") != "" {
 		return nil
 	}
-	fmt.Printf("Caching %s...", image)
 	dest := strings.Split(image, "/")
 	destName := fmt.Sprintf("/tmp/%s.tar", strings.Replace(strings.Join(strings.Split(dest[len(dest)-1], "/"), ""), ":", "-", -1))
+	fmt.Printf("Caching %s at %s...", image, destName)
 	if _, err := os.Stat(destName); os.IsNotExist(err) {
 		pull := p.PodmanNoCache([]string{"pull", image})
 		pull.Wait(90)
+		Expect(pull.ExitCode()).To(Equal(0))
```

Comment:
LGTM but since the interface of `CreateArtifact` specifies that it returns an `error`, and since there's only one caller, and that caller checks for error, would it be reasonable to explicitly check `ExitCode()` and return an error instead of using `Expect`? If not, would it make sense to redefine `CreateArtifact` as void with exit-on-error?

Reply:
Good catch, Ed! I opted for `void` and making `CreateArtifact` private. Thanks!